react/11/1/src/main.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useState } from "react";
+import ReactDOM from "react-dom";
+import Input from "./input";
+
+const MultiInput = () => {
+  const [inputData, setInputData] = useState("");
+
+  return (
+    <div>
+      <Input inputData={inputData} updateInputData={setInputData} />

```

Comment:
No need to create a separate component. Never write something you don't really need just to prepare for the future or because it seems more "Reacty" to do so:) Just write what you need, in this case, a list of inputs.